OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -118,7 +118,7 @@ public void UpdateMouseover()
 			}
 
 			var frozen = world.ScreenMap.FrozenActorsAt(world.RenderPlayer, worldRenderer.Viewport.ViewToWorldPx(Viewport.LastMousePos))
-				.Where(a => a.TooltipInfo != null && a.IsValid)
+				.Where(a => a.TooltipInfo != null && a.IsValid && (a.Actor != null && !a.Actor.IsInvisibleTo(world.RenderPlayer)))
```

Comment:
Wait a moment. That will change behavior. Frozen actors can be dead (then return null on Actor) so if I filter them here, then you can use the tooltip to find out if they are which would add another exploit.